hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -386,11 +389,14 @@ private void flushTransactions() {
               + "exit.", Thread.currentThread().getName());
         }
       } catch (IOException ex) {
-        terminate(ex);
+        terminate(ex, omResponse.get());
       } catch (Throwable t) {
-        final String s = "OMDoubleBuffer flush thread " +
-            Thread.currentThread().getName() + " encountered Throwable error";
-        ExitUtils.terminate(2, s, t, LOG);
+        StringBuilder s = new StringBuilder("OMDoubleBuffer flush thread " +
```

Comment:
Guess if `OzoneManagerDoubleBuffer#terminate` took exitCode as a parameter, you'd just need to replace the entire thing here with `terminate(ex, omResponse.get(), 2); `

Reply:
@duongkame Thank you for the suggestions, updated the patch.